src/components/task-plan/reading-stats.cjsx

```diff
@@ -42,6 +42,7 @@ Stats = React.createClass
 
     classes = 'reading-progress-bar'
     classes += ' no-progress' unless percent
+    classes += ' progress-bar-striped' if correctOrIncorrect is 'incorrect'
```

Comment:
Could/should this be `progress-bar-incorrect` to be more semantic (since the class adds the red gradient)